core/base/inc/Riostream.h

```diff
@@ -21,6 +21,10 @@
 //                                                                      //
 //////////////////////////////////////////////////////////////////////////
 
+#ifdef __APPLE__
+// ensures _FilesystemClock is defined first
```

Comment:
Can you put a link to the upstream issue here, and maybe also expand a bit on the comment so we will know in the future that it's a workaround and when we can remove it?